presto-main/src/test/java/com/facebook/presto/block/TestArrayBlock.java

```diff
@@ -158,6 +164,38 @@ public void testEstimatedDataSizeForStats()
         }
     }
 
+    @Test
+    public void testLogicalSizeInBytes()
+    {
+        int positionCount = 100;
+        int[] offsets = IntStream.rangeClosed(0, positionCount).toArray();
+        boolean[] nulls = new boolean[positionCount];
+
+        // Array(LongArrayBlock)
```

Comment:
consider replacing comments with variable names, e.g. 

* Block arrayOfLong = 
* Block arrayOfRleOfLong = 
* Block arrayOfRleOfArrayOfLong = 
...

Reply:
@mbasmanova I renamed the variables. However it's not as straightforward as the comment:
```
// Row(Dictionary(LongArrayBlock), Dictionary(Row(LongArrayBlock, LongArrayBlock)))
Block rowOfDictionaryOfLongAndDictionaryOfRowOfLongAndLong = ...
```
So I kept both the comments and renamed variables.